tests/conftest.py

```diff
@@ -14,7 +13,7 @@ def test_app():
 @pytest.fixture
 def enable_auth(monkeypatch):
     """Enable the authentication requirement for the API."""
-    monkeypatch.setattr("app.api.security.AUTH_ENABLED", True)
+    monkeypatch.setattr(settings, "auth_enabled", True)
```

Comment:
Is this still needed now? I thought with pytest-env we can now correctly control the env before each test is run

Reply:
Yes, I'd prefer to keep this fixture for explicitness and modularity. `pytest-env` allows us to define variables in the env once before any tests are run (in case we want to test anything related to actual *reading* of env vars for example), but we cannot dynamically _modify_ the values of any env vars once the tests begin.

Having these fixtures that explicitly override the *parsed* env var values at the per-test level ensures that the settings are predicable for tests that rely on specific configurations (e.g., tests for specific behaviour when auth is enabled, disabled, etc.). Otherwise, they would always rely on a specific global starting set of env var values, resulting in less flexibility and transparency of the tests.